x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
 import { EuiButton } from '@elastic/eui';
 
-import { CURRENT_MAJOR_VERSION } from '../../../../../common/version';
+import { docLinksService } from '../../../shared/doc_links/doc_links';
```

Comment:
nit: just import directly from the folder index.ts

```suggestion
import { docLinksService } from '../../../shared/doc_links'; 
```